modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -474,6 +488,89 @@ You can test whether the service is available at `_API_PLATFORM_HOST_/apiplatfor
 |===
 
 
+[[encrypt_password]]
+== Encrypt Passwords in an Existing mule-agent.yml File
+
+
+If you didn't set the `AGENT_VAR_master_password` environment variable when registering the agent 
+with Runtime Manager, you can encrypt these passwords later in the `mule-agent.yml` file:
+
+`keystorePassword`::
+Keystore password used to access the `mule-agent.jks` file
+`keystoreAliasPassword`:: 
+Keystore alias password used to access the `mule-agent.jks` file
+`password`::
+Password used for authenticating with a proxy server
```

Comment:
I know you dislike the output of dlist format, so how about...:

* `keystoreAliasPassword`: Keystore alias password used to access the `mule-agent.jks` file
* `password`: Password used for authenticating with a proxy server

This is a change I made to the SR regarding lists of items that have very short texts for descriptions.
I am currently working on a proposal to loosen up some current list and table styles, so stay tuned....